tests/unit/models/BucketInfo.spec.js

```diff
@@ -205,7 +220,8 @@ Object.keys(acl).forEach(
             true, undefined, testAzureInfo,
             testobjectLockEnabled,
             testObjectLockConfiguration,
-            testNotificationConfiguration);
+            testNotificationConfiguration,
+            testBucketTagging);
```

Comment:
```suggestion
            testBucketTagging,
        );
```
This helps a little with diffs and it makes the start and end on the same column

Reply:
First, thanks for your review, very helpful!
Done: https://github.com/scality/Arsenal/pull/1933/commits/08703718b843b40a802adb9aaafdf7840cd05baa#diff-31604c11f0c75e4caa3bd813de17aa4ff66eee7a586dd1a583acd1e1feaf48e6R224